app/services/search/query_builders/feed_content.rb

```diff
@@ -62,7 +86,7 @@ def term_keys
         TERM_KEYS.map do |term_key, search_key|
           next unless @params.key? term_key
 
-          { term: { search_key => @params[term_key] } }
+          { terms: { search_key => Array.wrap(@params[term_key]) } }
```

Comment:
`term` queries are for searching single values, [`terms` queries](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-terms-query.html#query-dsl-terms-query) are what you use when you want to search an array of values. The default search between items in the array is `OR`.